spark-bigquery-tests/src/main/java/com/google/cloud/spark/bigquery/integration/QueryPushdownIntegrationTestBase.java

```diff
@@ -16,12 +17,15 @@ public class QueryPushdownIntegrationTestBase extends SparkBigQueryIntegrationTe
 
   @Test
   public void testStringFunctionExpressions() {
+    BigQueryConnectorUtils.enablePushdownSession(spark);
```

Comment:
Instead of enabling and disabling pushdown manually for every test, why don't we do it in a @Before and @After method?

Reply:
addressed